sdk/identity/Azure.Identity/tests/CredentialTestHelpers.cs

```diff
@@ -39,12 +38,16 @@ public static (string Token, DateTimeOffset ExpiresOn, string Json) CreateTokenF
             return (token, expiresOn, json);
         }
 
-        public static (string Token, DateTimeOffset ExpiresOn, string Json) CreateTokenForAzureCliExpiresIn(int seconds = 30)
+        public static (string Token, string Json) CreateTokenForAzureCliExpiresOn(DateTimeOffset expiresOn, bool includeExpiresOn)
         {
-            var expiresOn = DateTimeOffset.UtcNow + TimeSpan.FromSeconds(seconds);
+            const string expiresOnStringFormat = "yyyy-MM-dd HH:mm:ss.ffffff";
+
+            var expiresOnString = expiresOn.ToLocalTime().ToString(expiresOnStringFormat);
             var token = TokenGenerator.GenerateToken(Guid.NewGuid().ToString(), Guid.NewGuid().ToString(), Guid.NewGuid().ToString(), Guid.NewGuid().ToString(), expiresOn.UtcDateTime);
-            var json = $"{{ \"accessToken\": \"{token}\", \"expiresIn\": {seconds} }}";
-            return (token, expiresOn, json);
+            var json = includeExpiresOn ?
+                $"{{ \"accessToken\": \"{token}\", \"expiresOn\": \"{expiresOnString}\", \"expires_on\": {expiresOn.ToUnixTimeSeconds()} }}" :
```

Comment:
Tip: embrace the string literals!!! :smile:

```suggestion
                $$"""{ "accessToken": "{{token}}", "expiresOn": "{{expiresOnString}}", "expires_on": {{expiresOn.ToUnixTimeSeconds()}} }" :
```

So much cleaner.